instaclustr/resource_cluster.go

```diff
@@ -401,18 +425,40 @@ func resourceClusterCreate(d *schema.ResourceData, meta interface{}) error {
 
 	clusterProvider.Tags = d.Get("tags").(map[string]interface{})
 
-	createData := CreateRequest{
+	var createData = CreateRequest{
 		ClusterName:           d.Get("cluster_name").(string),
 		Bundles:               bundles,
 		Provider:              clusterProvider,
```

Comment:
if we do the supporting of "multi dc with different provider" as per my comment below, this will need to be updated to handle that case